src/RTCZero.cpp

```diff
@@ -147,9 +147,13 @@ void RTCZero::standbyMode()
 {
   // Entering standby mode when connected
   // via the native USB port causes issues.
+	// Disable systick interrupt:  See https://www.avrfreaks.net/forum/samd21-samd21e16b-sporadically-locks-and-does-not-wake-standby-sleep-mode
```

Comment:
Please be consistent with the code style of the rest of the file and use a two space indent, not true tabs.
```suggestion
  // Disable systick interrupt:  See https://www.avrfreaks.net/forum/samd21-samd21e16b-sporadically-locks-and-does-not-wake-standby-sleep-mode
```